fsleyes_plugin_shimming_toolbox/st_plugin.py

```diff
@@ -464,8 +393,9 @@ def create_sizer(self):
 
     def add_button_run(self):
         """Add the run button which will call the ``Shimming Toolbox`` CLI."""
-        button_run = wx.Button(self.panel, -1, label="Run")
+        button_run = wx.Button(self.panel, -1, label="", size=(48, 48))
         button_run.Bind(wx.EVT_BUTTON, self.button_run_on_click)
+        button_run.SetBitmap(play_icon, dir=wx.TOP)
```

Comment:
This is most likely the problem for the run button being too high

Reply:
Added here: https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/40/commits/2cbc43ecfd2596198058bfdc461819c966abb8c9